Hudl.Mjolnir/Command/Command.cs

```diff
@@ -169,7 +177,8 @@ internal Command(string group, string name, string breakerKey, string poolKey, T
                 throw new ArgumentNullException("poolKey");
             }
 
-            if (defaultTimeout.TotalMilliseconds <= 0)
+            TimeoutsIgnored = IgnoreCommandTimeouts.Value;
+            if (!TimeoutsIgnored && defaultTimeout.TotalMilliseconds <= 0)
```

Comment:
Technically I don't think we need to check the ignored flag here - the default timeout's often going to get passed from a constant or hard-coded value, so it should be valid.

In fact, thinking about it more, I'd lobby to always validate the default timeout here, because otherwise, if timeouts are ignored, invalid default timeout values might go unnoticed in development.